__tests__/app.test.js

```diff
@@ -218,4 +218,46 @@ describe("GET /api/articles", () => {
       });
     });
   });
+})
+describe("GET  /api/articles/article_id/comments", () => {
+  test('GET 400: respond with error when article_id is not a number', () => {
+    return request(app)
+          .get("/api/articles/apple/comments")
+          .expect(400)
+          .then(({ body: {msg} }) => {
+            expect(msg).toBe('The article_id has to be a number');
+          });
+});
+  test("GET 200: respons with comment objects inside an array", () => {
+      return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body}) => {
+      expect(body.comments).toHaveLength(11)
+      expect(Array.isArray(body.comments)).toBe(true);
+      body.comments.forEach((comment)=> {
+       expect(comment).toEqual(
+       expect.objectContaining({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          article_id: 1
+       })
+       )
+      })
+      })
+  })
+  
+  test("GET 200: resones with an empty array when passed an valid article but has no comments in the database", () => {
+
+    return request(app)
+    .get("/api/articles/4/comments")
+    .expect(200)
+    .then(({body}) => {
+    expect(body.comments).toEqual([])
+
+    })
```

Comment:
I would add another 404 test for an article that doesn't exist. For example /api/articles/9999/comments

Reply:
Hi Rayhaan 
If I added the test shall I wait until my PR is approved?